libs/auth/scripts/configure_java_card.ts

```diff
@@ -0,0 +1,323 @@
+/* eslint-disable no-console */
+import { Buffer } from 'buffer';
+import { spawnSync } from 'child_process';
+import { assert, sleep } from '@votingworks/basics';
+import { Byte } from '@votingworks/types';
+
+import { CommandApdu, constructTlv } from '../src/apdu';
+import {
+  CARD_VX_ADMIN_CERT,
+  CARD_VX_CERT,
+  DEFAULT_PIN,
+  GENERIC_STORAGE_SPACE,
+  JavaCard,
+  MAX_INVALID_PIN_ATTEMPTS,
+  OPEN_FIPS_201_AID,
+  PUK,
+  VX_ADMIN_CERT_AUTHORITY_CERT,
+} from '../src/java_card';
+import {
+  construct8BytePinBuffer,
+  CRYPTOGRAPHIC_ALGORITHM_IDENTIFIER,
+} from '../src/piv';
+
+const APPLET_PATH = 'applets/OpenFIPS201-v1.10.2-with-vx-mods.cap';
+const GLOBAL_PLATFORM_PATH = 'scripts/gp.jar';
+
+/**
+ * CHANGE REFERENCE DATA ADMIN is an OpenFIPS201-specific extension of the PIV-standard CHANGE
+ * REFERENCE DATA command. It allows setting PINs and PUKs without the current value. APDUs must be
+ * sent over a GlobalPlatform Secure Channel.
+ */
```

Comment:
oh interesting. I want to flag that as something worth following up in our security review #3070

Reply:
For sure. Our modification to clear all PIN-gated keys on PIN reset applies to the admin path as well, but I'll be sure to verify that before I put up the OpenFIPS201 modification PR up for proper review (current draft here: https://github.com/votingworks/OpenFIPS201/pull/1)

Reply:
Created an [issue](https://github.com/votingworks/vxsuite-build-system/issues/10) in vxsuite-build for future support of the applet build.

Reply:
Awesome, thank you!